gitignore.txt

```diff
@@ -0,0 +1,398 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
File should be named .gitignore, otherwise it doesn't have effect and all files are still included in repo